com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/graph.py

```diff
@@ -109,7 +109,10 @@ def addOperator(self, kind, function=None, name=None, params=None, sl=None):
 
         if name is None:
             name = self._requested_name(None,action="Op", func = function)
-
+        
```

Comment:
If we have the following program:
```
topo = Topology("parallel_test")

s = topo.source(['1','2','3','4'])

s = s.parallel(5, name="CUSTOM").map(lambda x: -1*x).end_parallel()
s = s.map(lambda x: x)

s = s.parallel(5, name="CUSTOM").map(lambda x: -1*x).end_parallel()
s = s.map(lambda x: x)
```

It complains about the `CUSTOM` identifier being a duplicate.